test/GADM.jl

```diff
@@ -63,3 +63,63 @@ end
         @test diff < 0.01
     end
 end
+
+function getmeshespolygon(geometry)
+    # TODO refactor function
+    #converts [Float, Float] to Meshes Point object
+    topoint = x -> Meshes.Point(x)
+    coordinates = GeoInterface.coordinates(geometry)
+
+    if string(ArchGDAL.getgeomtype(geometry)) === "wkbMultiPolygon"
+        outer = map(topoint, first(coordinates[1]))
+        inner = []
+        if length(coordinates) > 1
+            for ring in coordinates[2:end]
+                push!(inner, map(topoint, first(ring)))
+            end
+        end
+        return Meshes.Polygon(outer, inner)
+    else
+        outer = map(topoint, first(coordinates))
+        return Meshes.Polygon(outer)
+    end
+end
+
+@testset "coordinates" begin
+    # invalid country code: lowercase
+    @test_throws ArgumentError GADM.coordinates("ind")    
+    # empty country code
+    @test_throws ArgumentError GADM.coordinates("")   
+    # invalid code other than format [A-Z]{3}
+    @test_throws ArgumentError GADM.coordinates("IND4") 
+    # Polygon
+    c = GADM.coordinates("VAT")
+    @test c isa Array{Array{Array{Array{Float64,1},1},1},1}
+    # MultiPolygon
+    c = GADM.coordinates("IND", "Gujarat")
+    @test c isa Array{Array{Array{Array{Float64,1},1},1},1}
+end
+
+@testset "meshes polygon" begin
+    # testing a complex polygon: State of Gujarat in India
+    gujarat = GADM.get("IND", "Gujarat")
+    gujarat_mp = getmeshespolygon(gujarat)
+
+    @test Meshes.issimple(gujarat_mp) == false
+    @test Meshes.hasholes(gujarat_mp) == true
+
+    orientation = Meshes.orientation(gujarat_mp)
+
+    @test orientation isa Tuple
+
+    # testing a simple polygon: Vatican City
+    vatican = GADM.get("VAT")
+    vatican_mp = getmeshespolygon(vatican)
+
+    @test Meshes.issimple(vatican_mp) == true
+    @test Meshes.hasholes(vatican_mp) == false
+
+    orientation = Meshes.orientation(vatican_mp)
+
+    @test orientation isa Symbol
+end
```

Comment:
I think it is better to remove the Meshes tests here and move them to Meshes.jl instead.